Plugins/FancyMesh/src/FancyMeshComponent.hpp

```diff
@@ -72,6 +73,9 @@ namespace FancyMeshPlugin
         const Ra::Core::Index* roIndexRead() const;
 
     private:
+        // the duplicate table for vertices, according to mesh data.
+        // Note: if duplicates have NOT been loaded, this is a 1-1 correspondance, i.e. m_duplicateTable[i] == i .
```

Comment:
For the sake of clarity, can you explain what I need to do if I want to iterate over the original mesh vertices (assuming I don't care about the loading options).
I guess this should be something like:

``` c++
for (uint id : *( component->getDuplicateTableOutput())) {
   auto & v = component->getVerticesRw()[id];
   // do stuff
}
```

If I want to skip the duplicates, I guess I need to do:

``` c++
for ( auto & v : *(component->getVerticesRw())) {
   // do stuff
}
```

2 questions:
 1. Can you please confirm this ?
 2. What about providing convenience iterators for this, instead of requiring the user to  work with the duplicate table ? This would make the API simpler to use and to understand for users. 
Bonus: in case we know in advance duplicates will not be loaded, both iterators could by-pass the duplicate table, since we know it's encoding the identity in that case.